src/modules/csm/state.py

```diff
@@ -150,19 +141,23 @@ def is_fulfilled(self) -> bool:
         return not self.unprocessed_epochs
 
     @property
-    def avg_perf(self) -> float:
-        """Returns average performance of all validators in the cache"""
-        return self.network_aggr.perf
+    def frame(self) -> tuple[EpochNumber, EpochNumber]:
+        if not self._epochs_to_process:
+            raise ValueError("Epochs to process are not set")
+        return min(self._epochs_to_process), max(self._epochs_to_process)
+
+    def get_network_aggr(self) -> AttestationsAccumulator:
+        """Return `AttestationsAccumulator` over duties of all the network validators"""
 
-    @property
-    def network_aggr(self) -> AttestationsAccumulator:
         included = assigned = 0
         for validator, acc in self.data.items():
             if acc.included > acc.assigned:
                 raise ValueError(f"Invalid accumulator: {validator=}, {acc=}")
             included += acc.included
             assigned += acc.assigned
-        return AttestationsAccumulator(
+        aggr = AttestationsAccumulator(
             included=included,
             assigned=assigned,
         )
+        logger.info({"msg": "Network attestations aggregate computed", "value": repr(aggr), "avg_perf": aggr.perf})
```

Comment:
Check aggr.perf will be dumped into json properly if it should be

Reply:
The state isn't dumped any more, so it's fine.